### ISSUE TITLE
Version not found at startup.

### ISSUE THREAD

**Issue by Magicrafter13** [jrbudda/Vivecraft_Spigot_Extensions#83]
Every time my server starts the plugin outputs:
```
[Vivecraft] Version not found. Are you from the future?
```
I didn't see any previous mention of this. Obviously it's not a big deal (not even a warning), but why is it here? Is something not working, or is this just "normal". (And making it go away would be nice.)

I'm using 1.16.5 r1. PaperMC.

**Comment by Techjar** [comment]
You sure you're using the 1.16.5 version of the plugin? You should only see that if you're using an older version, because we deleted the previous versions from the version.txt. I have now put them back though, so maybe you won't see it anymore. You should still make sure you're using 1.16.5 though.

**Comment by Magicrafter13** [comment]
![Untitled](https://user-images.githubusercontent.com/29127566/122345053-459e1300-cefc-11eb-98dc-90964c70c84c.png)
![Untitled](https://user-images.githubusercontent.com/29127566/122345164-66feff00-cefc-11eb-9113-7f020068c1fd.png)
Huh... welp. Looks like they do differ (though only by one byte? unless `cmp` stops at the first difference).

Time to restart my server with the old one deleted, and see if it still shows that.

It doesn't show that now - although I did almost point out that Vivecraft was detecting my server as being 1.16.4, which I verified it is not, it is indeed 1.16.5 - however this turned out to be a different plugin (that also starts with V). So *this* issue is resolved. Either jrbudda created a `1.16.5r1` jar file earlier that had a mistake in it (less likely), or I mistakenly renamed the file to `1.16.5r1.jar` when I unzipped (much more likely). Sorry if I wasted any time with this.

It wouldn't hurt to have the version number in the zipped file though :eyes:.
